### PR TITLE
Fix issue with running async tests in startup code.

### PR DESCRIPTION
**Pull request by crummel**
These caused AppX startup to time out and the test to fail.

### CODE REVIEW

**Comment by crummel** [comment]
cc @davidsh or @karajas for code review please.

File: src/xunit.runner.uwp/xunit.runner.uwp.csproj [hunk 1]
```diff
@@ -79,6 +79,9 @@
   <PropertyGroup Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
+  </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.CSharp.targets" />
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
 </Project>
```
Comment (davidsh) [inline src/xunit.runner.uwp/xunit.runner.uwp.csproj]: nit: missing newline at end of file.

**Comment by davidsh** [comment]
Left a comment but otherwise LGTM.

File: src/xunit.runner.uwp/xunit.runner.uwp.csproj (hunk 1 above)
Comment (crummel) [inline src/xunit.runner.uwp/xunit.runner.uwp.csproj]: Fixed.

**Comment by davidsh** [comment]
cc: @MattGal  - when this is merged in buildtools, we will need a change in CoreFx to pick this up.

**Comment by MattGal** [comment]
LGTM other than I'm unsure why you had to set a value for UseDotNetNativeToolchain... would expect this to only do interesting stuff for a test assembly build.

**Comment by davidsh** [comment]
@MattGal What sort of CoreFx changes are required to pick this up?

**Comment by MattGal** [comment]
@davidsh Just tweak the version string in the same two places as my PR did yesterday, but this time make the version end in 00808-02.  That should do it.

**Pull request merged**